tasks/update-granules-cmr-metadata-file-links/package.json

```diff
@@ -40,15 +40,15 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
-    "@cumulus/aws-client": "18.3.1",
-    "@cumulus/cmrjs": "18.3.1",
-    "@cumulus/common": "18.3.1",
+    "@cumulus/aws-client": "18.3.2",
+    "@cumulus/cmrjs": "18.3.2",
+    "@cumulus/common": "18.3.2",
     "@cumulus/cumulus-message-adapter-js": "2.2.0",
-    "@cumulus/distribution-utils": "18.3.1",
+    "@cumulus/distribution-utils": "18.3.2",
     "lodash": "^4.17.15"
   },
   "devDependencies": {
     "@cumulus/aws-client": "18.3.1",
```

Comment:
should this not also be brought up to 18.3.2?

Reply:
I suppose it should. Not sure why it's not bumped up as part of the update script.